actor/hello-build/.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build and test
+on:
+  pull_request:
+    branches: [main]
+    paths:
+      - "src/**"
+      - "Cargo.*"
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Add wasm32-unknown-unknown
```

Comment:
Nit: we should probably use the rust actions for this step, but isn't blocking because this is the same thing we have already

Reply:
sg, can refactor this later, just wanted to get it in so i can keep working on the wash build tests. This template won't be visible in `wash new actor` as it isn't in the favorites.toml file

Reply:
I actually chose to use the direct steps for clarity so that people could see exactly what they were doing, but I think using the rust action could be fine if there's a first-party one for GH